ci-operator/step-registry/ipi/conf/aws/ipi-conf-aws-commands.sh

```diff
@@ -13,31 +13,82 @@ CONFIG="${SHARED_DIR}/install-config.yaml"
 
 expiration_date=$(date -d '8 hours' --iso=minutes --utc)
 
+declare -g MASTER_FAMILY
+declare -g MASTER_TYPE
+
 function join_by { local IFS="$1"; shift; echo "$*"; }
 
+function get_master_size {
+  master_size=null
+  if [[ "${SIZE_VARIANT}" == "xlarge" ]]; then
+    master_size=8xlarge
+  elif [[ "${SIZE_VARIANT}" == "large" ]]; then
+    master_size=4xlarge
+  elif [[ "${SIZE_VARIANT}" == "compact" ]]; then
+    master_size=2xlarge
+  fi
+  echo ${master_size}
+}
+
+function set_master_type {
+  local size
+  master_type=null
```

Comment:
Why is this necessary?

Reply:
it's not. I just removed it, and replace the `${master_type}` to `${MASTER_TYPE}` used in this script. 👍🏼